controllers/common/ansible.go

```diff
@@ -73,8 +128,9 @@ func CreateAnsibleJob(policyAutomation *policyv1beta1.PolicyAutomation,
 		Resource: "ansiblejobs",
 	}
 
-	// Ansible tower API requires the lower case naming
-	ansibleJob.SetGenerateName(strings.ToLower(policyAutomation.GetName() + "-" + mode + "-"))
+	order := GetAnsibleJobOrder(policyAutomation, dynamicClient)
+	ansibleJob.SetGenerateName(strings.ToLower(policyAutomation.GetName() +
+		"-" + mode + "-" + strconv.Itoa(order) + "-"))
```

Comment:
Why is this change necessary?

Reply:
to sort easier, since we move checansiblejob it is ok to remove

Reply:
I tried to get the latest ansiblejob but tried to sort with time, but time in ansiblejob, the second is the smallest one. so hard to sort, ` creationTimestamp: "2023-04-27T18:56:04Z"` in  yaml file

Reply:
is there best way to sort by createtime